gnome-image-installer/gnome-image-installer.c

```diff
@@ -140,28 +140,60 @@ read_sanitized_string (const gchar *filename,
   return sanitized;
 }
 
+/**
+ * read_install_ini:
+ * @path: path to directory holding image files
+ *
+ * Reads install.ini within @path, if it exists, and applies the locale
+ * specified within, if any. This file is created by the winstaller
+ * when creating a reformatter USB, containing the locale that the winstaller
+ * itself was displayed in.
+ *
+ * Historically, this file was also used to configure unattended installations,
+ * but this functionality is now split out into a separate file. If a locale is
+ * specified both install.ini and unattended.ini, the latter takes precedence.
+ */
 static void
-read_keys (const gchar *path)
+read_install_ini (const gchar *path)
 {
-  gchar *ini = g_build_path ("/", path, "install.ini", NULL);
-  GKeyFile *keys = g_key_file_new();
+  g_autofree gchar *ini = g_build_path ("/", path, "unattended.ini", NULL);
```

Comment:
This actually reads `unattended.ini`, not `install.ini` as the name and documentation suggest.

Reply:
Good catch!